actions/workflows/ngi_uu_workflow.yaml

```diff
@@ -325,6 +326,28 @@ workflows:
             #    url: <% $.ngi_pipeline_url %>/<% $.runfolder_name %>
             ## END START NGI_PIPELINE ##
 
+            ## START CLEANUP ##
+
+            # this is a temporary workaround, the permanent fix is here: https://snpseq.atlassian.net/browse/ART-134
+            remove_from_scratch:
+              action: linux.rm
+              input:
+                hosts: <% $.host %>
+                private_key: <% $.biotank_user_key %>
+                recursive: true
+                target: /data/scratch/<% $.runfolder_name %>
+
+            move_to_archived:
+              with-items: folder in <% list($.runfolder, concat($.runfolder, "_archive")) %>
```

Comment:
it's because the archive workflow creates a `[runfolder_name]_archive` folder and we want to mark that for removal as well. as noted elsewhere, this functionality is a bit hacky in the context of this workflow and would make more sense in e.g. the archive service but I'm thinking that can wait for now

Reply:
:+1: